.changeset/busy-pandas-find.md

```
@@ -0,0 +1,6 @@
+---
+'@mastra/tanstack-start': patch
+'@mastra/next': patch
+---
+
+Update README to link new documentation
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)